litrpc/lit-accounts.proto

```diff
@@ -148,6 +154,28 @@ message UpdateAccountRequest {
     string label = 4;
 }
 
+message UpdateAccountBalanceRequest {
+    // The ID of the account to update. Either the ID or the label must be set.
+    string id = 1;
+
+    /*
+    The label of the account to update. If an account has no label, then the ID
+    must be used instead.
+    */
+    string label = 2;
+
+    /*
+    The balance update must either be an increase or a decrease of the balance
+    */
+    oneof update {
+        // Increase the account's balance by the given amount.
+        int64 add = 3;
+
+        // Deducts the given amount from the account balance.
+        int64 deduct = 4;
+    }
```

Comment:
when defining enums like this, it's good to ask: is there a future where more than 2 options will actually be needed? if not, then this can either be a boolean. Or we can even just have separate `Credit`/`Debit` definitions

Reply:
so either:

```
Update {
       - ID
       - Amount
       - Credit/Debit bool
}
```

or 

```
Credit { ID, Amount }
_and_
Debit {ID, Amount}
```

Reply:
> redit { ID, Amount }
> _and_
> Debit {ID, Amount}

Thanks! Went with the second option :)